src-electron/generator/generation-engine.js

```diff
@@ -742,11 +742,12 @@ async function generateAllTemplates(
   })
 
   // Next load the partials
-  packages.forEach((singlePkg) => {
+  for (const singlePkg of packages) {
     if (singlePkg.type == dbEnum.packageType.genPartial) {
-      templateEngine.loadPartial(hb, singlePkg.category, singlePkg.path)
+      const data = await fsPromise.readFile(singlePkg.path, 'utf8')
```

Comment:
I would keep loadPartial as an async function with path as an argument. We just can't run them with simple forEach.

Reply:
loadHelper is actually not an async function. The file read function is sometimes not finished before the template generation happens and partials are not loaded when this happens